### PR TITLE
add version for llama.cpp and ollama

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

add version for llama.cpp and ollama

### 4. How to test?

- [ ] Document test
image

image

### CODE REVIEW

File: docs/readthedocs/source/doc/LLM/Quickstart/llama_cpp_quickstart.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ See the demo of running LLaMA2-7B on Intel Arc GPU below.
 
 <video src="https://llm-assets.readthedocs.io/en/latest/_images/llama-cpp-arc.mp4" width="100%" controls></video>
 
+```eval_rst
+.. note::
+
+  Our current binary file version is consistent with `c780e75 <https://github.com/ggerganov/llama.cpp/commit/c780e75305dba1f67691a8dc0e8bc8425838a452>`_ of llama.cpp.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/llama_cpp_quickstart.md:12]: Our current version

File: docs/readthedocs/source/doc/LLM/Quickstart/ollama_quickstart.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ See the demo of running LLaMA2-7B on Intel Arc GPU below.
 
 <video src="https://llm-assets.readthedocs.io/en/latest/_images/ollama-linux-arc.mp4" width="100%" controls></video>
 
+```eval_rst
+.. note::
+
+  Our current binary file version is consistent with `v0.1.34 <https://github.com/ollama/ollama/releases/tag/v0.1.34>`_ of ollama.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/ollama_quickstart.md:12]: Our current version

Comment (rnwang04) [reply docs/readthedocs/source/doc/LLM/Quickstart/ollama_quickstart.md]: Done

File: docs/readthedocs/source/doc/LLM/Quickstart/llama_cpp_quickstart.md (hunk 1 above)
Comment (rnwang04) [reply docs/readthedocs/source/doc/LLM/Quickstart/llama_cpp_quickstart.md]: Done

**Review by sgwhat — APPROVED** [review]

**Pull request merged**